test/chrome/chromeUtils.test.ts

```diff
@@ -318,6 +318,27 @@ suite('ChromeUtils', () => {
                 chromeUtils.getMatchingTargets(targets, 'http://localhost'),
                 targets);
         });
+
+        test('handles double dots', () => {
+            const targets = makeTargets('http://localhost/app', 'http://localhost/site/../folder/../app');
```

Comment:
Sorry, I don't understand what you mean by "resolve". A double dot in a path means going up one level in the folder structure: `my/path/to/this/../resource` resolves to `my/path/to/resource`. 

That's precisely my use case (`file:///Users/baptiste/Documents/Projets/GitHub/bpesquet/thejsway-code/chapter02/js/../html/celsius_fahrenheit.html` should resolve to `file:///Users/baptiste/Documents/Projets/GitHub/bpesquet/thejsway-code/chapter02/html/celsius_fahrenheit.html`), and I think that's what my code does.